helm-charts/infisical/values.yaml

```diff
@@ -43,6 +43,13 @@ frontend:
     ## @param frontend.image.pullPolicy Backend image pullPolicy
     ##
     pullPolicy: IfNotPresent
+  ## container resource specifications [check the offical kubernetes documentations](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+  ##
+  resources:
+    limits:
+      memory: 100Mi
```

Comment:
Can you provide the metrics you used to come to this number?

Reply:
I've set these numbers based on the metrics we have from our infisical deployment on our production cluster. I will send you a screenshot in the DM.

Frontend is typically using 0.5mCPU & 75MB of Memory

The backend is using around 70mCPU & 140MB of Memory.